TombEngine/Scripting/Internal/TEN/Objects/Moveable/MoveableObject.cpp

```diff
@@ -601,6 +605,22 @@ void Moveable::SetRot(Rotation const& rot)
 	m_item->Pose.Orientation.z = ANGLE(rot.z);
 }
 
+/// Get the moveable's airborne status
+// @function Moveable:GetAirborne
+// @treturn (bool) true if entity animation is flying.
```

Comment:
I feel like this would be better worded as: "@return (bool) true if the entity is not grounded" or "@return (bool) true if the entity is in air"

but not sure, I would consult @Sezzary with English stuff :D

Reply:
I'm neither very convinced with "if flying", but IsAirborne is not exactly the grounded status.

Is a flag that creatures have, that indicate that status is "flying" (jumping floating, etc), so the physics tend to react accordingly. (like the gravity management).

However, is not a physical calculation of the grounded status. (For example, It gets activate during Monkey swing, although Lara is not on ground, she is not being affected by gravity).

I could write it in better way, although "not grounded" or "in air" doesn't fit either. @_@

Reply:
At the end I've updated with:

```
/// Get the moveable's airborne status
// @function Moveable:GetAirborne
// @treturn (bool) true if Lara state must react to aerial forces.
```
It may be the gravity, but other times may be also a force towards up like springboards. Or an backwards ejection like my case when she collides with the magic barrier. So... "aerial forces"?

Reply:
yeah should be good